scipy/stats/_continuous_distns.py

```diff
@@ -4242,6 +4247,43 @@ def _munp(self, n):
     def _entropy(self):
         return 2-np.log(2)
 
+    def fit(self, data, *args, **kwds):
+        if (isinstance(data, CensoredData)):
```

Comment:
This isn't what you want. Please consider the `_call_super_mom` decorator instead. I think that will do the right thing here.

Reply:
The generic test for censored data for all distributions only checks that `CensoredData` with very tight interval censoring and `CensoredData` with no actual censoring work approximately the same as regular arrays. There is not a check that `CensoredData` with more general censoring does something *different*. We don't need to add that now. With the new infrastructure I'll be working on shortly, this will be less of a concern because distributions won't be overriding the public `fit` method.